plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -160,15 +168,36 @@ class MultiContainerNetworkingFormSection extends mixin(StoreMixin) {
       address = `${hostname}${Networking.L4LB_ADDRESS}${port}`;
     }
 
+    const loadBalancerTooltipContent = (
+      <span>
+        {`Load balance the service internally (layer 4), and create a service
+        address. For external (layer 7) load balancing, create an external
+        load balancer and attach this service. `}
+        <a href="https://docs.mesosphere.com/1.8/usage/service-discovery/load-balancing-vips/"
```

Comment:
I'm a bit worried about release specific links being hard coded. I fear we might end up with links to wrong documentation for future releases. Can't think of a simple way to manage this though. Doc links could also change between releases. Any ideas?